ibu-imager/installation_configuration_files/scripts/installation-configuration.sh

```diff
@@ -28,6 +28,17 @@ echo "${RELOCATION_CONFIG_PATH} has been found"
 # Replace this with a function that loads values from yaml file
 set +o allexport
 
+function wait_for_etcd {
```

Comment:
We are going to rewrite all in golang i assume so maybe worth waiting for golang?